### ISSUE TITLE
Generate doc for IMS Collection release v1.3.0-beta.1 

### ISSUE THREAD

**Issue by ddgandhi** [IBM/z_ansible_collections_doc#53]
##### SUMMARY
Need to generate documentation for IMS Collection release v1.3.0-beta.1 

##### ISSUE TYPE
- Update Collection

**Comment by ddimatos** [comment]
There is an issue with the doc from the remote repository, the repository to pull dock from was not shared so I am using the tag, assuming that is correct:
```
  ibm_zos_ims:
    git: git@github.com:ansible-collections/ibm_zos_ims.git
    stable:
    latest: release-v1.3.0-beta.1
    origin: main
```

Then the issue is when doc is consumed by this repositories tooling an error occurred:
```
/Users/ddimatos/git/gh/z_ansible_collections_doc/source/reference/community.rst:14: WARNING: toctree contains reference to nonexisting document 'ibm_zos_ims/docs/source/community_guides'
looking for now-outdated files... none found
```
What this means is the `community_guides.rst` is not available under `release-v1.3.0-beta.1` and results in this error but also prevents all the other collections community guides from generating so its best to hold off on publishing documentation for IMS 1.3 beta.1.

The easies fix is to branch fro the tag `release-v1.3.0-beta.1`, lets call that `release-v1.3.0-beta.1-docfix`, fix the doc, then reply here so we can then pull from the branch `release-v1.3.0-beta.1-docfix` that includes `community_guides.rst`.

This is the current live view of the doc:
<img width="1071" alt="image" src="https://github.com/IBM/z_ansible_collections_doc/assets/25803172/46a46d8e-ae9c-4850-9861-7b6ef979a2d0">

This is how it looks now notice there is no IMS section:
<img width="565" alt="image" src="https://github.com/IBM/z_ansible_collections_doc/assets/25803172/7e32c23c-908a-4451-b10e-27a76d701f0d">

Since this is isolated to IMS , I am able to push the updated doc but you should plan to fix this in a new branch and open an isssue.

**Comment by itsBryantP** [comment]
Thanks @ddimatos. The missing community guidelines is fixed and the doc has been generated and published for the v1.3.0-beta.1 release.
